docs/pages/blog/2019-developer-survey-results.md

```diff
@@ -3,14 +3,9 @@ title: 2019 MUI Developer Survey results
 description: Results for the first yearly developer survey, 2019 edition.
 date: 2019-03-16T00:00:00.000Z
 authors: ['oliviertassinari', 'mbrookes']
-card: false
 tags: ['Developer survey']
 ---
 
-# 2019 MUI Developer Survey results
```

Comment:
This leads to duplicated h1 https://mui.com/blog/2019-developer-survey-results/, introduced in #30121